RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSeeds_cfi.py

```diff
@@ -52,3 +52,6 @@ def thresholds( wp ) :
 
 from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
 pp_on_AA.toModify(lowPtGsfElectronSeeds,MinPtThreshold = 5.0)
+
+from Configuration.Eras.Modifier_run3_upc_cff import run3_upc
+run3_upc.toModify(lowPtGsfElectronSeeds, ModelThresholds = thresholds("VL"), MinPtThreshold = 0.05)
```

Comment:
shouldn't everything egamma just follow `egamma_lowPt_exclusive` modifier ?

Reply:
Moved all the egamma related changes to egamma_lowPt_exclusive